src/gatling/scala/uk/gov/hmcts/reform/featuretoggle/actions/util/ToggleIdStore.scala

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.featuretoggle.actions.util
+
+import java.util.UUID
+
+import scala.collection.mutable
+import scala.util.Random
+
+object ToggleIdStore {
+
+  private val ids = mutable.SortedSet[String]()
```

Comment:
How about making this a mutable list? We wouldn't need to convert it to a Vector in order to retrieve a random element.

Reply:
done